### PR TITLE
use process_inverse_2018 [fix #26]

### LINKED ISSUE #26: process_inverse_mem should call process_inverse_2018 instead of process_inverse
**Describe the bug**

 process_inverse_mem should call process_inverse_2018 instead of process_inverse wich has been depreecated. See https://github.com/brainstorm-tools/brainstorm3/pull/743

### PR DESCRIPTION
**Pull request by Edouard2laire**
Update process_inverse_mem to call process_inverse_2018 instead of process_inverse. 

@rcassani, it this fix ok for you ?

### CODE REVIEW

File: processes/process_inverse_mem.m [hunk 1]
```diff
@@ -66,20 +66,23 @@
     
     % ===== GET OPTIONS =====
     % Default inverse options
-    OPTIONS = process_inverse('Compute');
+    OPTIONS = bst_inverse_linear_2018();
```
Comment (rcassani) [inline processes/process_inverse_mem.m:69]: Use `process_inverse_2018('Compute')` these are the options for computing the inverse.

`bst_inverse_linear_2018` are the options that are saved in the results file

**Review by rcassani — COMMENTED** [review]
This PR should do the trick.
I'd remove the extra empty lines to have a cleaner diff

File: processes/process_inverse_mem.m (hunk 1 above)
Comment (Edouard2laire) [reply processes/process_inverse_mem.m:69]: if i use 

then i get the following error: 

```matlab


***************************************************************************
** Error: [process_inverse_mem]  Sources > Compute sources: BEst
** Line 539: Unrecognized field name "NoiseMethod".
** 
** Call stack:
** >process_inverse_2018.m>Compute at 539
** >process_inverse_2018.m at 24
** >process_inverse_mem.m>Run at 142
** >process_inverse_mem.m at 24
** >bst_process.m>Run at 236
** >bst_process.m at 38
** >panel_process1.m>RunProcess at 138
** >panel_process1.m at 26
** >gui_brainstorm.m>CreateWindow/ProcessRun_Callback at 779
** >bst_call.m at 28
** >gui_brainstorm.m>@(h,ev)bst_call(@ProcessRun_Callback) at 299
** 
** 
** File: PA65/PA65_EPI-ekobayashi_20090720_04_600Hz_SingleTrial-f-CA/data_EEG_RF_spikes2_average_180815_2033.mat
** 
************

```

Comment (rcassani) [reply processes/process_inverse_mem.m]: (For completeness) 
This is solved in [7c9e3ea](https://github.com/brainstorm-tools/brainstorm3/commit/7c9e3ea07eb46f64346133b769e419fbe30ae878)

**Pull request merged**